lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4178,7 +4178,8 @@ AssignValue(CITE_UNIT           => undef);
 
 DefMacro('\@cite{}{}', '[{#1\if@tempswa , #2\fi}]');
 DefConstructor('\@@cite []{}', "<ltx:cite ?#1(class='ltx_citemacro_#1')>#2</ltx:cite>",
-  mode => 'text');
+  reversion => '\cite[#1]{#2}',
```

Comment:
Won't this leave an empty `[]` if there's no optional arg? `alias=>'\cite'` might be better?